src/diffpy/labpdfproc/tools.py

```diff
@@ -0,0 +1,7 @@
+def load_wavelength(args_wavelength, args_anode_type, WAVELENGTHS):
```

Comment:
I suggest to not pass in `WAVELENGTHS`.  define this as a global in tools.

Is there a reason that you don't just pass in `args` and then return `args`?  I think this would be more elegant.

Reply:
I think it's easier to write tests if we pass them separately? since we just need 2 parameters.

Reply:
Per your comment on another pull, "we need to load the correct wavelength into the args.wavelength variable even if this was not specified on the command line, and that i swhat the set_wavelength function will do."

We currently have the following codes related to wavelength:
wavelength = set_wavelength(...)
input_pattern = Diffraction_object(wavelength=wavelength)

Would it be better if I just replace wavelength with args.wavelength?